lldb/source/Core/Debugger.cpp

```diff
@@ -1958,6 +1984,12 @@ lldb::thread_result_t Debugger::DefaultEventHandler() {
   // are now listening to all required events so no events get missed
   m_sync_broadcaster.BroadcastEvent(eBroadcastBitEventThreadIsListening);
 
+  if (!m_statusline && GetShowStatusline())
+    m_statusline.emplace(*this);
```

Comment:
So, once it's enabled it can no longer be disabled? Could we send some sort of an event when the setting changes?